packages/booking/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@booking-services/booking",
+  "version": "1.0.0",
+  "description": "This package contains all bussiness logic from booking squad",
+  "author": "Booking Squad",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/main/packages/booking#readme",
+  "license": "ISC",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/artiko00/booking-services.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
```

Comment:
I noticed that you added the jest.config file, but you don't added the "test" command to run the tests.